Lib/test/test_pydoc/test_pydoc.py

```diff
@@ -693,6 +693,14 @@ def test_help_output_redirect(self):
         finally:
             pydoc.getpager = getpager_old
 
+    def test_lambda_return_annotation(self):
```

Comment:
Please add also a test for `__annotations__` without `return` key. Or with `return` key and with annotations for parameters.

Reply:
Done